gradle/libs.versions.toml

```diff
@@ -0,0 +1,135 @@
+[versions]
+accompanistSwiperefresh = "0.30.1"
+agp = "8.4.1"
+appcompat = "1.6.1"
+compiler = "1.4.4"
+constraintlayout = "2.1.4"
+converterMoshi = "2.9.0"
+coreKtxVersion = "1.5.0"
+coreTesting = "2.2.0"
+junitJupiter = "5.9.2"
+junitVintageEngine = "5.9.2"
+kluentAndroid = "1.72"
+kotlin = "1.8.10"
+coreKtx = "1.5.0"
+junit = "1.1.5"
+espresso = "3.5.1"
+kotlinxCoroutines = "1.6.4"
+#lifecycleRuntimeKtx = "2.8.1"
```

Comment:
Could commented versions and libraries be safely removed?

Reply:
I've removed commented libraries. However, we should have in mind that we need to update all libs/plugins (in a separate PR) and we should migrate to compose bom for versioning compose libs.